src/react-intl-messages.ts

```diff
@@ -616,6 +621,10 @@ export default {
         [FM.ACTIONSCORER_COLUMNS_ENTITIES]: 'Entities',
         [FM.ACTIONSCORER_COLUMNS_ISTERMINAL]: 'Wait',
         [FM.ACTIONSCORER_COLUMNS_TYPE]: 'Type',
+        
+        [FM.ACTION_DELETE_INUSE_TITLE]: 'Deleting actions used by train dialogs',
+        [FM.ACTION_DELETE_INUSE_PLACEHOLDER_PRESERVE]: 'This operation will preserves the reference to the action leaving a placeholder. All the dialogs using the action will become invalid.  This option can be useful to see where and how the action was used as you correct the dialogs.',
```

Comment:
Wondering if we might want to suggest that this option is useful if the intention is the Replace the action with another one, whereas the other option is for just removing the action?

Reply:
Yea, I'd meant for the "correct the dialogs" to be generic enough that you could fix it however you wanted, replace as one option but I added text at the end to say " or replace the action" so it's more clear.

I think we planned to have an actual replace action feature where you could select a source and target to do the swap, but that UI and feature still needs to be decided. This is interim solution to get ability to remove when you realize it was not needed and don't want to invalidate your dialogs.